### PR TITLE
 Add decision's validity models

### PR DESCRIPTION
**Pull request by catalinjitea**

### CODE REVIEW

File: ozone/core/models/process_agent.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    )
+
+    contain_technology = models.CharField(max_length=9999)
+
+    class Meta:
+        verbose_name_plural = 'process agent contain technologies'
+        db_table = 'pa_contain_technology'
+
+
+class ProcessAgentUsesValidity(Decision):
```
Comment (sorinstelian) [inline ozone/core/models/process_agent.py]: I think it's weird (from a user's perspective) to reference a decision like this.

We should have a foreign key to a decision here, I think.

File: ozone/core/models/process_agent.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+
+class ProcessAgentApplication(models.Model):
+    """
+    Applications of controlled substances as process agents, as approved
+    in table A of decision X/14 and updated periodically by the Meeting of the
+    Parties.
+    """
+
+    decision = models.ForeignKey(
```
Comment (sorinstelian) [inline ozone/core/models/process_agent.py]: This also seems weird. Foreign key?

File: ozone/core/models/process_agent.py (hunk 1 above)
Comment (catalinjitea) [reply ozone/core/models/process_agent.py]: Foreign key to decision seems like a better idea.

File: ozone/core/models/process_agent.py (hunk 2 above)
Comment (catalinjitea) [reply ozone/core/models/process_agent.py]: Here the foreign key is to ProcessAgentUsesValidity. Do you mean the field name?

**Pull request merged**